scripts/create_test_users.py

```diff
@@ -56,7 +54,6 @@
     ['Wyatt', datetime.date(1971, 3, 19), '450745912', [PUA_FULL_TIME, UI_PART_TIME], [8, 9]],
     ['Dunn', datetime.date(1997, 1, 4), '172089111', [PUA_FULL_TIME, UI_FULL_TIME, UI_PART_TIME], [5, 6, 7]],
     ['Neal', datetime.date(1939, 2, 8), '236990877', [PUA_FULL_TIME, UI_FULL_TIME, UI_PART_TIME], [5, 6, 7]],
-    # For internal use
```

Comment:
seems like it's still useful to designate some for internal use, since there'll be more UAT testing as we release things? i guess it doesn't matter what's in the codebase as long as it's updated in the google doc!

Reply:
Yeah, I think it's easier to update who's using what users in the Google doc as we're sending the information to various people.